clang/test/dpct/zero_length_array.cu

```diff
@@ -0,0 +1,29 @@
+// RUN: dpct --format-range=none -out-root %T/zero_length_array %s --cuda-include-path="%cuda-path/include"
+// RUN: FileCheck %s --match-full-lines --input-file %T/zero_length_array/zero_length_array.dp.cpp
+
+#include <stdio.h>
+
+// CHECK: void k(int *arr2) {
+// CHECK-NEXT:   /*
+// CHECK-NEXT:   DPCT1102:{{[0-9]+}}: Zero-length arrays are not permitted in SYCL device code.
+// CHECK-NEXT:   */
+// CHECK-NEXT:   int arr0[0];
+// CHECK-NEXT:   int arr1[1];
+// CHECK-NEXT:   // zero-sized local_accessor is allowed in SYCL
+// CHECK-NEXT: }
+__global__ void k() {
+  int arr0[0];
+  int arr1[1];
+  __shared__ int arr2[0];// zero-sized local_accessor is allowed in SYCL
```

Comment:
Comment should be updated to "// zero-sized local_accessor is not allowed in SYCL"